index.html

```diff
@@ -1,469 +1,716 @@
+                  <div class="header-txt">
+                    <p>Frameworks</p>
+                  </div>
+                  <button class="normal-btn">
+                    <svg
+                      width="24"
+                      height="25"
+                      viewBox="0 0 24 25"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                        fill="#505F79"
+                      />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </li>
+            <li>
+              <div class="main-frame">
+                <div class="frame-header">
+                  <div class="header-txt">
+                    <p>Skills</p>
+                  </div>
+                  <button class="normal-btn">
+                    <svg
+                      width="24"
+                      height="25"
+                      viewBox="0 0 24 25"
+                      fill="none"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        fill-rule="evenodd"
+                        clip-rule="evenodd"
+                        d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z"
+                        fill="#505F79"
+                      />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="contact">
+        <div class="content">
+          <div class="txt-holder">
+            <h1 class="primary-txt">Contact me</h1>
+            <p class="detail-txt">
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <div class="contact-form">
+            <form action="/contact-handling-form" method="post">
+              <ul>
+                <li>
+                  <input
+                    type="text"
+                    placeholder="Write your name here"
+                    value="Yeremias"
```

Comment:
Also, kindly remove the preset values for the input fields, cause they make the user experience bad since I have to delete the content of the input field before I add my own. Also, all the input fields should have the attribute `required`. Kindly update the same. The user name field has a max length of 30 characters or so. Kindly update it also.